cartridges/int_adyen_overlay/cartridge/scripts/adyenGetConnectedTerminals.ds

```diff
@@ -0,0 +1,74 @@
+/**
+* Send request to adyen to get connected terminals based on merchant account and storeId
+*
+*/
+
+//script include
+var AdyenHelper = require("*/cartridge/scripts/util/AdyenHelper");
+var Logger = require("dw/system/Logger");
+
+function getTerminals() {
+    try {
+        var requestObject = {};
+
+        var callResult = null,
+            resultObject = null,
+            service = AdyenHelper.getService(AdyenHelper.SERVICE.CONNECTEDTERMINALS);
+
+        if (service == null) {
+            Logger.getLogger("Adyen").error("Could not retrieve connected terminals, error creating the service");
+            return;
```

Comment:
This could throw an error instead, so that you would have a centralized logging code.